src/main/java/org/teamresistance/frc/util/Controllers.java

```diff
@@ -0,0 +1,601 @@
+package org.teamresistance.frc.util;
+
+import edu.wpi.first.wpilibj.Joystick;
+
+
+/**
+ * This utility class assists in obtaining values accurately from
+ * controls. Controller pointers must be configured before use using.
+ * Any Controllers objects not being used should be set to null. <p>
+ *  <code>Controllers.set(driver360, operator, null);</code> or <br>
+ *  <code>Controllers.set(driver360, operator1, operator2);</code>  or <br>
+ *  <code>Controllers.set(driverRight, driverLeft, operator1, operator2);</code>
+ *
+ * @author Bryce Matsumori
+ * @author Ben Clark
+ * @author Patrick Wamsley
+ *
+ * @see edu.wpi.first.wpilibj.Joystick
+ *  ThresholdHandler
+ *
+ */
+public final class Controllers {
```

Comment:
This class is for the XBox controller, right? If you run a search-all in IntelliJ and type `XBox`, you should see there's an `XBoxController` class already in the WPILib, so this utility isn't necessary.